### PR TITLE
Add nearby petplays view

### PR DESCRIPTION
**Pull request by tony-gomes**
DO NOT MERGE THIS!

- Add PetPlays Index View Template
- Plus NearbyPetPlays logic in partial
- Add PetPlays Schema,Controller,Model
- Add Search#Index route

### CODE REVIEW

File: spec/features/search/user_can_see_nearby_petplays_spec.rb [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+    petplay3 = PetPlay.create(title: 'PetPlay Test 3',
+                              pet_players: "#{petplayer3A.id}, #{petplayer3B.id}, #{petplayer3C.id}",
+                              location: '80202')
+
+    visit '/petplays'
+    expect(current_path).to eql('/petplays')
+
+    # within find(".content") do
+      # expect(page).to have_css('h2.content-head')
```
Comment (tony-gomes) [inline spec/features/search/user_can_see_nearby_petplays_spec.rb:64]: I am having some difficulty with these capybara within blocks... maybe ya'll can spot what I am missing... @team

Comment (StephanieFriend) [reply spec/features/search/user_can_see_nearby_petplays_spec.rb:64]: I pulled it down into my own branch and gave it a few tweaks. If you want to take a look at 'testing_tonys_branch' maybe that'll help a bit. :)

File: app/views/shared/_nearby_pet_plays_mvp.html.erb [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  <div id="pet-play-<%= index + 1 %>">
+    <h2 class="pet-play-head"><%= pet_play.title %></h2>
+
+    <div class="pet-players-wrapper">
+      <% list_pet_players(pet_play.pet_players).each do |player| %>
+        <p class="player-name"><%= player %></p>
+      <% end %>
+    </div>
+  </div>
+<% end %>
```
Comment (StephanieFriend) [inline app/views/shared/_nearby_pet_plays_mvp.html.erb:13]: Nice making a partial! We just have to make sure it is used elsewhere as well

File: spec/features/search/user_can_see_nearby_petplays_spec.rb (hunk 1 above)
Comment (tony-gomes) [reply spec/features/search/user_can_see_nearby_petplays_spec.rb:64]: Dope.

Comment (tony-gomes) [reply spec/features/search/user_can_see_nearby_petplays_spec.rb:64]: P.S. GH emojis are wack...

**Comment by tony-gomes** [comment]
- Add Stephanie's improvements... check
- All tests pass

**Pull request merged**